ci/check_style.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION
```

Comment:
Borrowing from https://github.com/rapidsai/rapids-cmake/blob/branch-23.12/ci/check_style.sh, but simplified. We could add dependencies.yaml and use the same script as the rest of RAPIDS if desired.
```suggestion
# Copyright (c) 2023, NVIDIA CORPORATION

set -euo pipefail

pip install pre-commit

# Run pre-commit checks
pre-commit run --all-files --show-diff-on-failure
```